extension/code.js

```diff
@@ -205,23 +205,23 @@ function halloween() {
 			let levels = [
 				`<span style='font-size: 50px; display: block; margin-bottom: 20px;'>🎃</span><h1>Stage 1, Introduction</h1>
 				Ho ho ho! Wait, it's Halloween, not Christmas... whoops! <em>Mwahahahahaha</em>.. that's better. I'm <a href='/users/DrHalloweenMonster/' target="_blank">Dr. Halloween-Monster</a>, and I'd love to help you learn to code in a fun and creative way! At the end, I'll give you a special code that will make your profile have a picture of me (🎃) next to your username, and I'm sure you'll get a lot out of it. So, are you with me?
-				<button style="display: block; margin: 5px auto;" onclick='localStorage.setItem("iOHalloween", JSON.stringify({stage: 1, messagesRead: false})); location.reload();'>Yes, of course I'm with you Dr. Halloween-Monster!</button> (Note, check the nav bar -- if I'm underlined pink, that means you have an unread notification from me)
+				<button style="display: block; margin: 5px auto;" onclick='localStorage.setItem("iOHalloween", JSON.stringify({stage: 1, messagesRead: false})); location.reload();'>Yes, of course I'm with you Dr. Halloween-Monster!</button> (Note, check the nav bar -- if I'm underlined pink, that means you have an unread notification from me... actually you should have one now if you clicked the button... if you didn't click the button... the CLICK IT!)

```

Comment:
Remove the definite article "the" before "CLICK IT" -- so it makes sense.

Otherwise, thanks for the pull request this is amazing!

Reply:
Fixed that!